web/src/components/Internal.tsx

```diff
@@ -0,0 +1,8 @@
+import * as React from 'react'
+import { Tag } from './Tag'
+
+const color = '#0b62f5'
+
+export function Internal() {
```

Comment:
Maybe just write a small comment what is this for, standalone like this it is very hard to say what it is.
You could also name it "InternalTag".